src/Jackett.Common/Indexers/Torrentseeds.cs

```diff
@@ -240,6 +258,14 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                     release.PublishDate = parsedDateTime;
                     release.Grabs = ParseUtil.CoerceInt(qGrabs.TextContent);
                     release.Files = ParseUtil.CoerceInt(qFiles.TextContent);
+
+                    var qImdb = row.QuerySelector("td.torrent-table-name > div > p.float-down > span.label-warning > a");
+                    if (qImdb != null)
+                    {
+                        var deRefUrl = qImdb.GetAttribute("href");
+                        release.Imdb = ParseUtil.GetImdbID(WebUtility.UrlDecode(deRefUrl).Split('/').Last());
+                    }
+                    
```

Comment:
My editor says there's spaces in this blank line, which is against whitespace guidelines.

Reply:
Tried to resolve this, have not had my editor saying this.

Reply:
That took care of it.